scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1159,10 +1156,7 @@ bool antistall_consume(s32 cpu, struct cpu_ctx *cctx)
 	consumed = false;
 	zero = 0;
 
-	if (!enable_antistall)
-		return false;
-
-	if (!cctx || !cctx->layer_idx || !cpu)
```

Comment:
nice catch, so this means even if the antistall cpu doesn't have a layer it can still try to consume stalled out tasks.